### DISCUSSION TITLE
Update EE error/message template on upgrade from EE2/3, if it hasn't been modified

### DISCUSSION THREAD

**Discussion by intoeetive** [ExpressionEngine/ExpressionEngine#2864] [Feature Requests]
In EE6 we have updated the specialty templates (error / message / site offline) to have better styles, more in line with CP styles.

However when sites are upgrading, we don't touch Specialty Templates, because people might have made modifications to those.

This ends up with EE7 sites using old grey error templates, which do not look nice these days.

The proposal is on upgrade to compare existing error/message templates with those that existed in EE1/2/3/4/5 and if it's exactly the same, have it replaced with the modern one.

**Comment by TomJaeger** [comment]
Love it!
